jest-puppeteer.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  launch: {
+    args: ['--no-sandbox', '--disable-setuid-sandbox'],
```

Comment:
Can we add a comment explaining why we need this?

E.g.: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#setting-up-chrome-linux-sandbox